src/report.rs

```diff
@@ -267,7 +267,7 @@ impl From<RawData> for Report {
             report.add_raw_text(format!(
                 "{} <b><a href=\"https://{}.subscan.io/validator/{}\">{}</a></b>",
                 is_active_desc,
-                data.network.name.to_lowercase().trim().replace(" ", ""),
+                data.network.name.to_lowercase().trim().replace(" ", "-"),
```

Comment:
The immediate patch looks good to me, hoever all of this looks very fragile. 

If the name becomes "Creditcoin 3" then the URL will be incorrect again. 

Q: where does the name come from? Is if hard-coded somewhere in the source code?